ChangeLog.md

```diff
@@ -1,5 +1,15 @@
 ## Gerbera - UPnP AV Mediaserver.
 
+### v1.8.0
+- Significant improvement in import speeds with large collections
+- Handle Samsung X_SetBookmark action, for saving position of viewed media, when enabled by flag
+- Extract from mp3 with ID3v1 tags, not just v2
+- Add a retry when binding to interfaces, should make gerbera much more reliable to start with slow network startup.
+- Hack around broken libstdc++ large file support on Debian 10 with 32-bit OS (Mostly older ARM devices)
+- Fix build with GCC11
+- Fix debian package to work out of the box
```

Comment:
Add search support for containers, i.e. albums, artists and genres in virtual layout